contracts/lockdrop/src/contract.rs

```diff
@@ -1447,12 +1447,14 @@ pub fn callback_deposit_liquidity_in_astroport(
         }
     }
 
+    coins.sort_by(|a, b| a.denom.cmp(&b.denom));
+
     cosmos_msgs.push(CosmosMsg::Wasm(WasmMsg::Execute {
```

Comment:
maybe let's move this to some provide_liquidity function, which will always sort the coins so that we don't forget to do it in some other place?

Reply:
Maybe, but it is required for any executions of instantiative and executive methods with many coins

Reply:
lgtm, can i merge ?

Reply:
yes ser